draft-ietf-quic-transport.md

```diff
@@ -2536,11 +2531,10 @@ mentions a stream ID.  In this state, the endpoint MUST observe advertised
 stream and connection data limits (see {{flow-control}}).
 
 A stream transitions from this state to "closed" by completing transmission of
-all data.  This includes sending all data carried in STREAM frames up including
-the terminal STREAM frame that contains a FIN flag.
+all data.  This includes sending all data carried in STREAM frames up to and
```

Comment:
nit: I would just remove "up" and say "This includes sending all data carried in STREAM frames including the terminal STREAM frame that contains a FIN flag."